frontend/src/i18n/de-DE.json

```diff
@@ -49,17 +52,27 @@
 
   "fetchError.title": "Abruf von Daten fehlgeschlagen",
 
-  "createVault.title": "Tresor erstellen",
-  "createVault.description": "Das Vault-Admin-Passwort sollte geheim gehalten werden und wird nur für administrative Zwecke benötigt. Bitte achte darauf, dass du dir das Passwort merkst.",
-  "createVault.vaultName": "Tresorname",
-  "createVault.vaultDescription": "Beschreibung",
-  "createVault.password": "Vault-Admin-Passwort",
-  "createVault.password.description": "Muss mindestens 8 Zeichen enthalten.",
-  "createVault.passwordConfirmation": "Vault-Admin-Passwort bestätigen",
-  "createVault.passwordConfirmation.passwordsMatch": "Passwörter stimmen überein!",
-  "createVault.passwordConfirmation.passwordsDoNotMatch": "Passwörter stimmen nicht überein",
-  "createVault.submit": "Tresor erstellen",
+  "createVault.enterRecoveryKey.title": "Tresor wiederherstellen",
+  "createVault.enterRecoveryKey.description": "Gib den Recovery-Key deines bestehenden Tresors ein, um daraus eine neue Tresorvorlage zu erstellen.",
```

Comment:
In Cryptomator, the recovery key is called "Wiederherstellungsschlüssel" in the German translation. We should consider either renaming the hub translation or using "Recovery Key" as a specific, non-translatable term.

Reply:
I'll give it a try again. The problem was that the word is just sooo looong. 😄